node_cli/core/node_options.py

```diff
@@ -0,0 +1,74 @@
+#   -*- coding: utf-8 -*-
+#
+#   This file is part of node-cli
+#
+#   Copyright (C) 2022 SKALE Labs
+#
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU Affero General Public License as published by
+#   the Free Software Foundation, either version 3 of the License, or
+#   (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+from filelock import FileLock
+
+from node_cli.utils.helper import read_json, write_json, init_file
+from node_cli.configs.node_options import NODE_OPTIONS_FILEPATH
+
+logger = logging.getLogger(__name__)
+
+
+class NodeOptions:
```

Comment:
Why do we need lock here? Is this file changed from somewhere else?

Reply:
it may be changed in admin (though, it's read-only in sync-admin now) - so just in case